src/frequenz/client/common/pagination/pagination.py

```diff
@@ -0,0 +1,100 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Module to define the pagination used with the common client."""
+
+from __future__ import annotations  # required for constructor type hinting
+
+import logging
+from dataclasses import dataclass
+
+# pylint: disable=import-error, no-name-in-module
+from frequenz.api.common.v1.pagination_info_pb2 import (
+    PaginationInfo as PBPaginationInfo,
+)
+
+# pylint: disable=import-error, no-name-in-module
+from frequenz.api.common.v1.pagination_params_pb2 import (
+    PaginationParams as PBPaginationParams,
+)
+
+# Set up logging
+_logger = logging.getLogger(__name__)
```

Comment:
Because this one is unused, and since it's a *private* member of the module (starts with `_`), this might probably fail a check in `flake8` or `pylint`.

```suggestion
```